PostmanCollections/VTEX - Recommendations BFF API.json

```diff
@@ -0,0 +1,1079 @@
+                  "key": "Content-Type",
+                  "value": "text/html"
+                }
+              ],
+              "body": "Recommendations BFF is running!",
+              "cookie": []
+            }
+          ],
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "4005fc53-2d88-47f1-a5af-48c7694e15cd",
+                "type": "text/javascript",
+                "exec": [
+                  "// Validate status 2xx \npm.test(\"[GET]::/api/recommend-bff/healthcheck - Status code is 2xx\", function () {\n   pm.response.to.be.success;\n});\n",
+                  "// Validate if response header has matching content-type\npm.test(\"[GET]::/api/recommend-bff/healthcheck - Content-Type is text/html\", function () {\n   pm.expect(pm.response.headers.get(\"Content-Type\")).to.include(\"text/html\");\n});\n"
+                ]
+              }
+            }
+          ],
+          "protocolProfileBehavior": {
+            "disableBodyPruning": true
+          }
+        }
+      ],
+      "event": []
+    },
+    {
+      "id": "006702a1-6959-42a8-9bb7-a21a0da3455a",
+      "name": "Events",
+      "description": {
+        "content": "",
+        "type": "text/plain"
+      },
+      "item": [
+        {
+          "id": "f763dd64-cb77-4c8f-96e9-0ea254969cef",
+          "name": "Recommendation view",
+          "request": {
+            "name": "Recommendation view",
+            "description": {
+              "content": "Records when a user views a recommendation.\n\n## Permissions\n\nThis endpoint does not require [authentication](https://developers.vtex.com/docs/guides/authentication) or [permissions](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+              "type": "text/plain"
+            },
+            "url": {
+              "path": [
+                "api",
+                "recommend-bff",
+                "events",
+                "recommendation-view",
+                "v2"
+              ],
+              "host": [
+                "{{baseUrl}}"
+              ],
+              "query": [
+                {
+                  "disabled": false,
+                  "description": {
+                    "content": "(Required) VTEX account name. Must match the account name in the `x-vtex-rec-origin` header.",
+                    "type": "text/plain"
+                  },
+                  "key": "an",
+                  "value": "apiexamples"
+                }
+              ],
+              "variable": []
+            },
+            "header": [
+              {
+                "disabled": false,
+                "description": {
+                  "content": "(Required) Required format: `{accountName}/{sourceType}/{appOrIntegrationId}`. Must match the `an` query parameter for account name.",
+                  "type": "text/plain"
+                },
+                "key": "x-vtex-rec-origin",
```

Comment:
One thing that I've noticed now and I forgot to change on our swagger is that this field is actually required in all endpoints. Can you add to the others here while I change on our side?